test-requirements.txt

```diff
@@ -1 +1 @@
-mock; python_version < '3.3'
\ No newline at end of file
+mock; python_version < '3.3'
```

Comment:
I can't help but wonder what has changed here?

Reply:
Must be a newline ending or something similar...